cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -302,10 +307,10 @@ def _validate(self, instance_type, efa_enabled, gdr_support):
 class EfaPlacementGroupValidator(Validator):
     """Validate placement group if EFA is enabled."""
 
-    def _validate(self, efa_enabled, placement_group_id, placement_group_enabled):
-        if efa_enabled and not placement_group_id and not placement_group_enabled:
+    def _validate(self, efa_enabled, placement_group_enabled):
+        if efa_enabled and not placement_group_enabled:
             self._add_failure(
-                "You may see better performance using a placement group for the queue.", FailureLevel.WARNING
+                "Using a placement group is required for EFA-enabled compute resources.", FailureLevel.ERROR
```

Comment:
here we don't want to require the usage of a placement group, but we want to require the PlacementGroup setting to be explicitly included in the config. Also we need to extend PlacementGroup accepted values in order to include not only dynamic and id but also a keyword to explicitly disable it (disabled?).

The message here should point out the fact that you need to explicitly disable placement groups with PlacementGroup: disabled if needed.

Reply:
PlacementGroup is not a single parameter; it's a section:

```yaml
PlacementGroup:
  Enabled: Boolean  # true
  Id: String
```

I see what you're saying about requiring an explicit value as opposed to requiring the use of a placement group, and how that would in turn affect the right error message to display to users. Instead of extending the valid values for the PlacementGroup section's Enabled field, why not do this by instead ensuring that PlacementGroup's Enabled field is not implicit when EFA is enabled? This would let us to continue defining the Enabled field as a Bool rather than a Str field where we need to convert to a bool.

Reply:
yes totally. I had completely forgotten that now we have the Enabled boolean field.